deps/zlib-1.2.11/.gitignore

```diff
@@ -1,2 +1,4 @@
 zconf.h.included
-cmake-build*
\ No newline at end of file
+cmake-build
+zconf.h
```

Comment:
The zconf.h and Makefile are part of the zlib source and removing it may leads to build failure AFAIK.

Reply:
Removing the Makefile should be fine since the build script now uses Makefile.in instead. I am curious as to why you removed zconf.h though

Reply:
zconf.h is not part of code base in the first place.

Reply:
zconif.h is part of the code base: https://github.com/madler/zlib
Anyway, let's wait for the test result and see. I remembered it failed on Windows once I removed them.

Reply:
zconf.h.cmakein      zconf.h template for cmake
zconf.h.in                 zconf.h template for configure
Yeah let's wait for the results.